package.json

```
@@ -3,7 +3,7 @@
   "version": "0.5.0",
   "description": "Testing console the right way",
   "engines": {
-    "node": "^10 || ^12 || ^13 || ^14"
+    "node": "^10 || ^12 || ^13 || ^14 || ^15"
```

Thanks for the PR! I'm thinking we should probably loosen the guards. Maybe just:

```suggestion
    "node": "^10 || >=12"
```

---

Yeah good idea, pushed an update with your suggestion 👍